addons/mail/static/tests/chatter_tests.js

```diff
@@ -197,16 +197,31 @@ QUnit.test('chatter is not rendered in mode === create', function (assert) {
     assert.strictEqual(form.$('.o_chatter').length, 1,
         "chatter should be displayed");
 
+    // entering create mode
     form.$buttons.find('.o_form_button_create').click();
+    assert.strictEqual(form.$('.o_chatter').length, 1,
+        "chatter should still be displayed in create mode");
 
-    assert.strictEqual(form.$('.o_chatter').length, 0,
-        "chatter should not be displayed");
+    // topbar buttons disabled in create mode (e.g. 'send message')
+    form.$('.o_chatter_topbar button').each(function () {
```

Comment:
or you could do something like 
`assert.strictEqual(form.$('.o_chatter_topbar button:not(:disabled)').length, 0, ...)`